coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRuntime.java

```diff
@@ -717,7 +717,7 @@ private void load() {
                             }
                         } catch (Throwable ex) {
                             log.error("Failed to load metadata from {} with epoch {} due to {}.",
-                                tp, epoch, ex.toString());
+                                tp, epoch, ex.toString(), ex);
```

Comment:
Because the whole exception `ex` is logged, changing `ex.toString()` to `ex.getMessage()` would eliminate a lot of redundant information in the log